src/pet.js

```diff
@@ -0,0 +1,71 @@
+const MAX_FITNESS_LEVEL = 10;
+const MIN_HUNGER_LEVEL = 0;
+function Pet(name) {
```

Comment:
You could definitely add more constant values here, for example:
```
const MIN_FITNESS = 0; 
const AGE_INCREMENT = 1;
const FITNESS_INCREMENT = 4;
const HUNGER_INCREMENT = 3;
const HUNGER_THRESHOLD = 5
const FITNESS_THRESHOLD = 3
const MAX_HUNGER = 10
const MAX_AGE = 30
```
Also, you should have a clear line between the constant definitions and the function definition